symbolic_pymc/meta.py

```diff
@@ -284,13 +268,35 @@ def _repr_pretty_(self, p, cycle):
                     p.pretty(obj)
 
 
+@dispatch(type)
+def _metatize(obj):
```

Comment:
Add a docstring describing how this handles type objects by returning an existing meta type/class for the given type, or by creating a new meta type/class for the type.